JavaScript/Hashnode_Scripts/script.js

```diff
@@ -0,0 +1,71 @@
+// all requires
+const fetch = require("node-fetch");
+const ObjectsToCsv=require("objects-to-csv");
+
+
+//variables required
+var page=0;
+var post_count=process.argv[2];
+
+
+
+// Working Function
+fetchPosts=async(post_count)=> {
+    var posts_data=[]; // stores posts
+    var count=0;
+    var current_post_count=0;
+
+    while(current_post_count<post_count)
+    {
+        const query=`{
+            storiesFeed(type:BEST
+                page:${page}){
+                title
+                brief
+                author{
+                  name
+                }
+              }
+        }`
+
+        const response = await fetch('https://api.hashnode.com', {
+            method: 'POST',
+            headers: {
+                'Content-type': 'application/json',
+            },
+            body: JSON.stringify({ query }),
+        })
+    
+        const ApiResponse=await response.json();
+        
+        var posts=ApiResponse.data.storiesFeed;
+
+        current_post_count+=posts.length;
+
+        page++;
+
+        for(var post of posts){
+            count++;
+
+            var post_object={}
```

Comment:
In JavaScript, generally the variable conventions generally followed are TitleCase or camelCase. Replace all the Underscore_case variable names with appropriate ones. Just a small enhancement, pardon for being too picky haha with best practices!

Reply:
Changes Done, Please Have a Look.